domains/cocainepotato.json

```diff
@@ -6,7 +6,7 @@
         },
     
         "record": {
-            "CNAME": "https://cocaine-potato.github.io/"
+            "CNAME": "cocaine-potato.github.io"
```

Comment:
Little to no content on site.

Reply:
content will come once i got a domain up, on the site its very outdated and will be drastically changed once i got a domain to work with